e2e/cypress/integration/messaging/MM-T134-visual-verification-of-tooltips_spec.js

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
```

Comment:
Currently we are following a format where the name of the file does not contains the test ID. This is because a single file can contain multiple tests. Hence can you rename the file and remove test ID. Also we are following a `_` instead of `-` in filenames.

Reply:
Thank-you. Change made and I will follow this format going forward.